Cargo.toml

```diff
@@ -13,6 +13,10 @@ finite automata and guarantees linear time matching on all inputs.
 """
 categories = ["text-processing"]
 
+[badges]
+travis-ci = { repository = "rust-lang/regex" }
+appveyor = { repository = "rust-lang-libs/regex" }
```

Comment:
I took the name from the CI badge that you already have in the README, and for some reason the name is different on AppVeyor compared to TravisCI.